client/src/components/LinkDisplay/index.js

```diff
@@ -54,12 +54,7 @@ const LinkDisplay = ({ content }) => {
           </button>
         </div>
       </div>
-      <div
-        className={`${styles.pinDisplay} ${darkMode ? styles.darkOpenLink : styles.lightOpenLink}`}
-      >
-        <span className={styles.pinValidMsg}>PIN (valid for 30 minutes)</span>
-        <PinDisplay content={content.pin} />
```

Comment:
You can also remove `PinDisplay` from import

Reply:
No problem, done.

Reply:
I can also delete the PinDisplay component folder

Reply:
lets leave it for now